projects/hslayers/src/components/data/catalogue/add-data-catalogue.html

```diff
@@ -19,7 +19,7 @@ <h6 class="dropdown-header">{{'ADDDATA.CATALOGUE.filterType' | translate}}</h6>
                     </div>
                 </div>
             </div>
-            <input type="search" class="form-control w-50" [placeholder]="'SEARCH.search' | translate" name="search"
+            <input type="search" class="form-control w-50" [placeholder]="'PANEL_HEADER.SEARCH' | translate" name="search"
```

Comment:
You can Use also COMMON.search instead of PANEL_HEADER.SEARCH, just so You know. :)

Reply:
I already changed this in develop, please rebase. It got broken at some point and we didn't have proper text there.